pandas/core/generic.py

```diff
@@ -6017,16 +6017,17 @@ def __finalize__(self, other, method: str | None = None, **kwargs) -> Self:
                 object.__setattr__(self, name, getattr(other, name, None))
 
         if method == "concat":
-            objs = kwargs["objs"]
```

Comment:
Slightly more performant to define objs=other.objs once?

Reply:
Can we get rid of kwargs? Will this mess up for subclasssss?

Reply:
> Slightly more performant to define objs=other.objs once?

Sure thing. Make sense

> Can we get rid of kwargs? Will this mess up for subclasssss?

Looks generally unused but not exactly sure the scope of how this is used. I'd prefer to maybe do this in a separate change